INTERNALS.md

```diff
@@ -230,9 +230,33 @@ generates the following JSON object describing the change:
 
 
 Applying operations to the local state
------------------------------------
-
-TODO
+--------------------------------------
+
+The local state is maintained as a map with the following keys:
+* `states`: A map
```

Comment:
It seems to me that you're sending along a WIP commit as the PR. From your comment before, I concluded you would send incrementally complete doc requests. I'd suggest sending a small PR any time you have even a useful paragraph or two and we can merge them, or if you have questions about how/why something is the way it is, send those separately.

I mention all this because I'd love to hit "merge" on the first paragraph below, but this not-terribly useful-in-its-current-form list of things above it, I'd rather not.

Reply:
@pvh fair enough! I've just updated the PR to remove the list!